app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/localService.ts

```diff
@@ -0,0 +1,157 @@
+/** @file Module containing the API client for the local backend API.
+ *
+ * Each exported function in the {@link Backend} in this module corresponds to an API endpoint. The
+ * functions are asynchronous and return a `Promise` that resolves to the response from the API. */
+import * as cloudService from './cloudService'
+import * as newtype from '../newtype'
+import * as platformModule from '../platform'
+import * as projectManager from './projectManager'
+
+// ========================
+// === Helper functions ===
+// ========================
+
+function ipWithSocketToAddress(ipWithSocket: projectManager.IpWithSocket) {
+    return newtype.asNewtype<cloudService.Address>(`ws://${ipWithSocket.host}:${ipWithSocket.port}`)
+}
+
+// ===============
+// === Backend ===
+// ===============
+
+interface CurrentlyOpenProjectInfo {
+    id: projectManager.ProjectId
+    project: projectManager.OpenProject
+}
+
+export class Backend implements Partial<Omit<cloudService.Backend, 'platform'>> {
```

Comment:
No docs - there should be clear info of what a "backend" is, as this is not a name we are using in all other parts of the project.